tests/test_sklearn_voting_regressor_converter.py

```diff
@@ -0,0 +1,70 @@
+"""Tests VotingRegressor converter."""
+
+import unittest
+import numpy
+from sklearn import datasets
```

Comment:
This is no longer used, can you remove it? I am surprised flake8 check didn't fail.

Reply:
Done